### ISSUE TITLE
Que algunas swords, shields y rings se pongan al darles use

### ISSUE THREAD

**Issue by armadaazteca** [Armada-Azteca/ArmadaAztecaServer#213]
Para poder poner en hotkey algunos items para poder equiparlos super fast mientras andas en los putazos.

**Comment by Ponid** [comment]
No seria mas fácil actualizar a una versión mas adelante? 10.xx

**Comment by Eduardox** [comment]
Si se actualiza hay pedos con cooldowns y eso no?

**Comment by Ponid** [comment]
Porque abría pedos con cooldowns o.o? Viéndolo desde el punto de vista de players (que sinceramente es lo único que les importa), todos (absolutamente todos) los bots que los players usan actualmente funcionan en 10.xx+ (que al actualizar desde 8.6 esta fue la queja principal) y al actualizar se podrían usar mas bots todavía. (Windbot, Xenobot, iBot). Y otra sugerencia que vi por ahí fue poner cast system (que la verdad si hace falta).

**Comment by armadaazteca** [comment]
Cambiar de versión si es un pedo mayor.  Tengo muchas cosas en sscripts que no son compatibles,  y esta característica es fácil de implementar con sscript digo yo.  Y bajar de versión esta el pedo de que se pueden perder algunas cosillas,  como mounts.  Y no hay mucho código por ahí de versiones tan viejas.  No recuerdo a partir de que versión hay un cambio importante en los containers a partir de esa sería lo difícil.  Estos dias he estado bien clavado en madres de bots,  estoy aprendiendo y aunque no he avanzado en prototipos en sí,  yo digo que si podría avanzar a un punto en el que las funciones principales de bot me la pelearán. Y podré hacerlas yo al 100. Voy a abrir un issue sobre eso en estos días

**Comment by Aron154** [comment]
Wind bot y ibot son la misma cosa....

**Comment by Ponid** [comment]
@Aron154 Nope, ya no. iBot continuo siendo desarrollado, aparte de Windbot.
